service/src/main/java/org/apache/celeborn/server/common/service/store/db/mapper/ClusterInfoMapper.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.server.common.service.store.db.mapper;
+
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Select;
+
+import org.apache.celeborn.server.common.service.model.ClusterInfo;
+
+public interface ClusterInfoMapper {
+
+  @Insert(
+      "INSERT INTO celeborn_cluster_info(name, namespace, endpoint, gmt_create, gmt_modify) "
+          + "values (#{name}, #{namespace}, #{endpoint}, #{gmtCreate}, #{gmtModify})")
```

Comment:
UPCASE all SQL keywords
```suggestion
          + "VALUES (#{name}, #{namespace}, #{endpoint}, #{gmtCreate}, #{gmtModify})")
```